Core/Frameworks/Baikal/Model/Structs/LDAPConfig.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Baikal\Model\Structs;
+
+/**
+ * Struct that holds the Configuration parameters for LDAP authentication.
+ *
+ * @author El-Virus <elvirus@ilsrv.com>
+ * @license http://sabre.io/license/ Modified BSD License
```

Comment:
Baikal is GPL, not BSD. Having one file with a different license feels weird..

Reply:
My bad. Fixed.